server/chart_config.json

```diff
@@ -679,41 +677,38 @@
   },
   {
     "category": "Crime",
-    "title": "Total Crime",
+    "title": "Total crime",
     "statsVars": ["Count_CriminalActivities_CombinedCrime"],
     "isOverview": true,
     "relatedChart": {
-      "title": "Total Crime Rate",
+      "title": "Crimes per 100,000 people",
```

Comment:
Since the numerator is #crimes and denominator is #people, % does not make sense. I chose 100k since it is large enough for most places and commonly used:
![image](https://user-images.githubusercontent.com/18621425/96799944-0c3af500-13b9-11eb-9b82-821760d6c1e0.png)